File: show-case/jpa-show-case/src/main/java/com/yicj/study/jpa/entity/UserInfo.java
package com.yicj.study.jpa.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.ToString;

/**
 * <p>
 * UesrInfo
 * </p>
 *
 * @author yicj
 * @since 2024/09/04 13:47
 */
@Data
@ToString(callSuper = true)
@TableName("user_info")
public class UserInfo extends BaseEntity {

    @TableField("username")
    private String username ;

    @TableField("password")
    private String password ;

    @TableField("email")
    private String email ;

    @TableField("phone")
    private String phone ;

    @TableField("address")
    private String address ;
}

File: show-case/mybatis-show-case/src/main/java/com/yicj/study/mybatis/entity/UserInfo.kt
package com.yicj.study.mybatis.entity

import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import lombok.Data
import lombok.ToString

@Data
@TableName("user_info")
class UserInfo(
    @TableId var id: String? = null,
    var username: String? = null,
    var password: String? = null,
    var address: String? = null) {
    override fun toString(): String {
        return "UserInfo(id=$id, username=$username, password=$password, address=$address)"
    }
}